test/functional/apps/dashboard/_dashboard_filtering.js

```diff
@@ -266,17 +272,33 @@ export default function ({ getService, getPageObjects }) {
         await dashboardExpect.pieSliceCount(1);
       });
 
+      it('Removing filter pills and query unfiters data as expected', async () => {
+        await dashboardPanelActions.clickEdit();
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        await renderable.waitForRender();
+        await queryBar.setQuery('');
+        await queryBar.submitQuery();
+        await filterBar.removeFilter('sound.keyword');
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        await dashboardExpect.pieSliceCount(5);
+
+        await PageObjects.visualize.saveVisualization('Rendering Test: animal sounds pie');
+        await PageObjects.header.clickDashboard();
+
+        await dashboardExpect.pieSliceCount(5);
+      });
+
       it('Pie chart linked to saved search filters data', async () => {
         await dashboardAddPanel.addVisualization('Filter Test: animals: linked to search with filter');
-        await dashboardExpect.pieSliceCount(3);
+        await dashboardExpect.pieSliceCount(7);
```

Comment:
Because I added a test to remove the saved filter above.  I did this as clean up, so the test suite leaves the code just as it was when it started.  I wanted to run the suite muiltiple times in a row and If I didn't do this, it'd fail the second time around because the filter is then actually saved on the instance (good or bad, I'm not reloading all the index data after each test suite to save time).